pkg/sql/codegen_elasticdl.go

```diff
@@ -279,10 +285,17 @@ func elasticDLPredict(w *PipeWriter, pr *extendedSelect, db *DB, cwd string, ses
 
 	// Create and execute ElasticDL prediction command
 	cmd := elasticdlPredictCmd(cwd, modelDefFilePath, predictFiller)
-	cmd.Stdout = cw
-	cmd.Stderr = cw
-	if e := cmd.Run(); e != nil {
-		return fmt.Errorf("code %v failed %v", modelDefCode, e)
+	stderr, _ := cmd.StderrPipe()
+	if err := cmd.Start(); err != nil {
+		log.Fatal(err)
+	}
+
+	scanner := bufio.NewScanner(stderr)
+	for scanner.Scan() {
+		fmt.Println(scanner.Text())
```

Comment:
Why `fmt.Println`? Shouldn't we send the logs to the user via `cw`?

Reply:
Thanks. Fixed.